lib/fixed-points.nix

```diff
@@ -341,13 +353,34 @@ rec {
       in fApplied // g final prev';
```

Comment:
`prev'` could be `gPrev`, or it could be inlined, because the name doesn't really add anything, except an unnecessary pointer in the heap.